tasks/winbuildscripts/Generate-Chocolatey-Package.ps1

```diff
@@ -8,6 +8,14 @@ $releasePattern = "(\d+\.\d+\.\d+)"
 $releaseCandidatePattern = "(\d+\.\d+\.\d+)-rc\.(\d+)"
 $develPattern = "(\d+\.\d+\.\d+)-devel\+git\.\d+\.(.+)"
 
+$installMethod = "online"
+$nuspecFile = c:\mnt\chocolatey\datadog-agent-online.nuspec
+$nupkgs = Get-ChildItem c:\mnt\chocolatey\datadog-agent*.msi
+if (($nupkgs | Measure-Object).Count -gt 0) {
```

Comment:
We could have three cases here:
- `count == 0`: offline
- `count == 1`: online
- `count > 1`: error, abort